nautobot_golden_config/api/views.py

```diff
@@ -91,3 +95,13 @@ class ConfigReplaceViewSet(CustomFieldModelViewSet):  # pylint:disable=too-many-
     queryset = models.ConfigReplace.objects.all()
     serializer_class = serializers.ConfigReplaceSerializer
     filterset_class = filters.ConfigReplaceFilterSet
+
+
+class ConfigToPushViewSet(ReadOnlyModelViewSet):  # pylint:disable=too-many-ancestors
+    """Detail REST API view showing configuration to push to appliances."""
+
+    permission_classes = [IsAuthenticated]
```

Comment:
Need to consider the permissions of `GoldenConfig` as well

Reply:
hopefully is already addressed